### PR TITLE
Feat hero

### PR DESCRIPTION
**Pull request by BillyG83**
- responsive hero i built on the plane to Dubai
- takes text and bg image
- has custom text color for title and sub title
- can have option sub title
- custom bg color
- takes child components

### CODE REVIEW

File: src/styles/StyledElements.ts [hunk 1]
```diff
@@ -3,6 +3,17 @@ import color from './colors';
 import fonts from './fonts';
 import resetCSS from './reset';
 
+export const H1Styled = styled.h1`
```
Comment (Y0moo) [inline src/styles/StyledElements.ts:6]: Nice PR @BillyG83. Why you didn't use `<Typography>` instead of creating a new H1 style

**Comment by locothedev** [comment]
Do not forget to import/export in `src/index.ts`

**Comment by BillyG83** [comment]
@locothedev i fixed the index TY

@Y0moo i now use Typography TY, i think i might have fixed you `argentumSans` issue too, so easy to miss. Too deep in the code mines sometime, right =]

File: src/styles/StyledElements.ts (hunk 1 above)
Comment (BillyG83) [reply src/styles/StyledElements.ts:6]: done TY, i'll leave this in for now and clean up all the `StyledElements` later if thats ok

**Review by BillyG83 — COMMENTED** [review]
big fan of my own work xD

**Pull request merged**